exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers = [...numbers, 4];
```

Comment:
It's a good way of doing it but it says "don't change the array literal for this practice, could you think of other ways to assign it?